act/headings.md

```diff
@@ -0,0 +1,137 @@
+# Headings
+
+## Rule Identifier
+
+gel-foundations-headings
```

Comment:
What system/pattern will you use to create these identifiers? Does `gel-foundations` refer to something specific, like the github project by that name? <https://github.com/bbc/gel-foundations> I don't think Headings are part of the `gel-foundations` project.

Reply:
It's nothing more complex than 'gel-', followed by either 'foundations' or 'component' as the sections on https://bbc.github.io/gel/, followed by the pattern name, and so has nothing to do with a project. I think the last two parts make sense, but the 'gel' bit could be more specific to separate it from other parts of GEL documentation.